hostinfo/base.lua

```diff
@@ -16,12 +16,14 @@ limitations under the License.
 
 local Object = require('core').Object
 local vutils = require('virgo/utils')
+local virgo_misc_utils = require('virgo/util/misc')
 
 --[[ HostInfo ]]--
 local HostInfo = Object:extend()
 function HostInfo:initialize()
   self._params = {}
   self._error = nil
+  self._util = virgo_misc_utils
```

Comment:
you won't want to cache the module here... 

Take a look at the gist I sent... Maybe subclass Hostinfo to be a HostInfoStdoutProcess